### PR TITLE
Updates to Readme

### PR DESCRIPTION
**Pull request by aroltsch**
1. Separate command by line otherwise it looks as a one-liner

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,8 +68,10 @@ You'll need to install the libraries required to connect to the VACOLS Oracle da
 
 3) Install via Homebrew:
 
+```
```
Comment (aroltsch) [inline README.md]: Without this change it looks like this: 
image

File: README.md [hunk 2]
```diff
@@ -20,6 +20,7 @@ Install the base dependencies via Homebrew:
     brew install rbenv nodenv yarn
     brew tap ouchxp/nodenv
     brew install nodenv-nvmrc
+    brew install postgresql
```
Comment (aroltsch) [inline README.md]: We need to install here otherwise `bundle install` won't succeed without PG config file.

File: README.md [hunk 3]
```diff
@@ -119,14 +130,16 @@ This should install clean. If you have errors, try ... FIXME.
 
 ### Setup the development Postgres user
 
-Add these to your `.bashrc`:
+Add these to your `.bash_profile`:
```
Comment (aroltsch) [inline README.md]: bashrc didn't work for me

File: README.md [hunk 4]
```diff
@@ -39,6 +40,14 @@ Once you've done that, close your terminal window and open a new one. Verify tha
...
 If you don't see both, stop and debug.
 
+### Install Git
+
+```
+brew install git
+```
+
+We are using 2-factor authentication with Github so, for example, when you access a repository using Git on the command line using commands like `git clone`, `git fetch`, `git pull` or `git push` with HTTPS URLs, you must provide your GitHub username and your personal access token when prompted for a username and password. Follow directions [here](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) on how to do that.
+
```
Comment (ferlatte) [inline README.md]: git comes with the xcode tools that get installed above (which you need for bundle install to work also); you don't need to install it separately if you don't want to. Do we need it from brew for some other reason?

File: README.md [hunk 5]
```diff
@@ -119,14 +130,16 @@ This should install clean. If you have errors, try ... FIXME.
...
+Add these to your `.bash_profile`:
 
 ```
 export POSTGRES_HOST=localhost
 export POSTGRES_USER=postgres
 export POSTGRES_PASSWORD=postgres
 ```
 
+(Reload the file `source ~/.bash_profile`)
+
```
Comment (ferlatte) [inline README.md]: Depending on what else is in there, this isn't a safe operation. Better to tell folks to close their terminal and make a new one.

File: README.md (hunk 2 above)
Comment (mdbenjam) [reply README.md]: Really? That sucks :/

File: README.md (hunk 5 above)
Comment (mdbenjam) [reply README.md]: Oh really? Why might this not be safe? Is this for non idempotent things in there?

**Comment by ferlatte** [comment]
There can be; I’ve had it be an issue on other projects, so I tend to document that folks should just start a new terminal to ensure a clean and debugable environment.

It’s probably not that important.

M

**Pull request merged**